docs/tutorials/openfoam/spack-openfoam.md

```diff
@@ -0,0 +1,296 @@
+# HPC Toolkit - Install and Run OpenFOAM on a Slurm Cluster
+
+HPC Toolkit is an open-source software offered by Google Cloud which makes it
+easy for customers to deploy HPC environments on Google Cloud.
+
+In this tutorial you will use the HPC Toolkit to:
+
+* Deploy a [Slurm](https://github.com/SchedMD/slurm-gcp#readme) HPC cluster on
+  Google Cloud
+* Use [Spack](https://spack.io/) to install the OpenFOAM application and all of
+  its dependencies
+* Run a [OpenFOAM](https://www.openfoam.com/) job on your newly provisioned
+  cluster
+* Tear down the cluster
+
```

Comment:
Here, I would maybe add the following lines at the bottom.

"Deploying the cluster and installing the applications dependencies may take 2 hours and 34 minutes if no spack cache is provided. With a complete spack cache, this time can be reduce to 6 minutes." (please check this time).

Reply:
Done. PTAL as final was slightly different that what you suggested.